### ISSUE TITLE
issue in creating lora

### ISSUE THREAD

**Issue by leonviz87** [cocktailpeanut/fluxgym#296]
Tried installing it and to train a lora but it just keep coming up with this issue and not even a lora trained

[2025-01-12 15:59:20] [INFO] Running C:\Users\loveh\fluxgym\outputs\mandy-chinese\train.bat

[2025-01-12 15:59:20] [INFO]

[2025-01-12 15:59:20] [INFO] C:\Users\loveh\fluxgym>accelerate launch --mixed_precision bf16 --num_cpu_threads_per_process 1 sd-scripts/flux_train_network.py --pretrained_model_name_or_path "C:\Users\loveh\fluxgym\models\unet\flux1-dev.sft" --clip_l "C:\Users\loveh\fluxgym\models\clip\clip_l.safetensors" --t5xxl "C:\Users\loveh\fluxgym\models\clip\t5xxl_fp16.safetensors" --ae "C:\Users\loveh\fluxgym\models\vae\ae.sft" --cache_latents_to_disk --save_model_as safetensors --sdpa --persistent_data_loader_workers --max_data_loader_n_workers 2 --seed 42 --gradient_checkpointing --mixed_precision bf16 --save_precision bf16 --network_module networks.lora_flux --network_dim 4 --optimizer_type adafactor --optimizer_args "relative_step=False" "scale_parameter=False" "warmup_init=False" --split_mode --network_args "train_blocks=single" --lr_scheduler constant_with_warmup --max_grad_norm 0.0 --learning_rate 8e-4 --cache_text_encoder_outputs --cache_text_encoder_outputs_to_disk --fp8_base --highvram --max_train_epochs 16 --save_every_n_epochs 4 --dataset_config "C:\Users\loveh\fluxgym\outputs\mandy-chinese\dataset.toml" --output_dir "C:\Users\loveh\fluxgym\outputs\mandy-chinese" --output_name mandy-chinese --timestep_sampling shift --discrete_flow_shift 3.1582 --model_prediction_type raw --guidance_scale 1 --loss_type l2

[2025-01-12 15:59:20] [INFO] 'accelerate' is not recognized as an internal or external command,

[2025-01-12 15:59:20] [INFO] operable program or batch file.

[2025-01-12 15:59:20] [ERROR] Command exited with code 1

[2025-01-12 15:59:20] [INFO] Runner: <LogsViewRunner nb_logs=6 exit_code=1>

Anyone has any idea what is this issue? and solutions?
